### ISSUE TITLE
Adding loading png files into Succulent module

### ISSUE THREAD

**Issue by vcvrackideas** [Miserlou/RJModules#56]
Hey, your sense of humour is nice! I talk about Succulent\Meow module. But it would be great if you would add a file loader for user pngs file. I just test random picture, named it as Blank_20HP, and replace succulent picture in res folder -  it was been cropped of course but works for me.

**Comment by Miserlou** [comment]
Yeah! I was thinking that there'd just be a new module called Image or PNG or something like that since people seem to want that, but I just need to figure out if I can dynamically resize a module to match the width..
